planner/core/physical_plans.go

```diff
@@ -268,6 +268,8 @@ type PhysicalHashJoin struct {
 
 	Concurrency     uint
 	EqualConditions []*expression.ScalarFunction
+
+	OuterHashJoin bool
```

Comment:
1. s/ OuterHashJoin/ UseOuterToBuild ?
I think `OuterHashJoin` is not explicit.
2. Add a comment for this attribute.

Reply:
ok